invoice.go

```diff
@@ -90,6 +90,8 @@ type Invoice struct {
 	DueDate             int64             `json:"due_date"`
 	EndingBalance       int64             `json:"ending_balance"`
 	Forgiven            bool              `json:"forgiven"`
+	HostedInvoiceURL    string            `json:"hosted_invoice_url"`
+	InvoicePDF          string            `json:"invoice_pdf"`
```

Comment:
Sorry for the nit, but I think this should be below `ID` for it to line up alphabetically.